blogposts/2021/show-us-your-calendar.md

```diff
@@ -4,151 +4,163 @@ description: 'Engineers are leading the way toward rethinking the 40 hour work w
 
 ## Jean du Plessis, director of engineering, has a nonlinear workday
 
-[Jean du Plessis](https://about.sourcegraph.com/handbook/company/team#jean-du-plessis-he-him), a director of engineering at Sourcegraph, considers his calendar “an experimentation over a long period of time.” Located in South Africa, Jean has reports all over Europe, Asia, and both coasts of North America. “I have a big time frame to cover,” he says.
+[Jean du Plessis](https://about.sourcegraph.com/handbook/company/team#jean-du-plessis-he-him), a director of engineering at Sourcegraph, considers his calendar "an experimentation over a long period of time." Located in South Africa, Jean has reports all over Europe, Asia, and both coasts of North America. "I have a big time frame to cover," he says.
 
-The nonlinear workday, [according to GitLab](https://about.gitlab.com/company/culture/all-remote/non-linear-workday/), the company who popularized the term, “decouples time from work and acts as a forcing function to embrace asynchronous workflows.”
+The nonlinear workday, [according to GitLab](https://about.gitlab.com/company/culture/all-remote/non-linear-workday/), the company that popularized the term, "decouples time from work and acts as a forcing function to embrace asynchronous workflows."
 
 ![Nonlinear calendar graphic](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Nonlinear.png)
 
 For Jean, this means using the morning to take care of the chores that most would do after work, such as grocery shopping and walking the dog. From the late morning to the afternoon, he focuses on meetings with his teammates and completing work leftover from the previous day.
 
-> “I’ve found myself playing video games in the afternoon and I’ve found myself meeting up with friends.”
-> Jean du Plessis, director of engineering
+> "I've found myself playing video games in the afternoon and I've found myself meeting up with friends." - Jean du Plessis, director of engineering
 
-In the afternoon, Jean is frank: he might just nap. 3:00 pm (his time) is when most people in the east coast of the United States start to wake up, so the next three hours have some of the best overlap for meetings. From 6:00 pm to 8:30 pm is family time: “We have supper together, kids get bathed, we read a story.” And from 8:30 pm to 10:00 pm, he handles any stray follow-up meetings and completes some light work.
+In the afternoon, Jean is frank: he might just nap. 3 pm (his time) is when most people in the east coast of the United States start to wake up, so the next three hours have some of the best overlap for meetings. From 6:00 pm to 8:30 pm is family time: "We have supper together, kids get bathed, we read a story." And from 8:30 pm to 10:00 pm, he handles any stray follow-up meetings and completes some light work.
```

Comment:
Can you make this line clearer? It reads as if Jean chooses to nap in the afternoon at the exact time when there is meeting overlap... I think you mean before teammates come online but it's a bit confusing!